src/App.tsx

```diff
@@ -186,37 +187,31 @@ export default function App() {
             .join('')};
         </div> */}
       {choices?.length > 0 && (
-        <div
-          ref={scrollRef}
+        <SimpleBar
+          scrollableNodeProps={{ ref: scrollRef }}
+          className="px-4 pb-4 flex flex-col text-black dark:text-white w-full max-h-screen focus:border-none focus:outline-none outline-none"
```

Comment:
I added [simplebar](https://www.npmjs.com/package/simplebar-react) because scroll view is such a crucial part of the GUI and native one looks rather ugly.